### PR TITLE
senpai: init at 2021-05-27

### PR DESCRIPTION
**Pull request by malte-v**
###### Motivation for this change
[senpai](https://sr.ht/~taiite/senpai/) is an IRC client designed for use with [soju](https://sr.ht/~emersion/soju/).

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [21.11 Release Notes](./CONTRIBUTING.md#generating-2111-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/applications/networking/irc/senpai/default.nix [hunk 1]
```diff
@@ -0,0 +1,38 @@
+{ lib, buildGoModule, fetchFromSourcehut, installShellFiles, scdoc }:
+
+buildGoModule rec {
+  pname = "senpai";
+  version = "2021-05-27";
```
Comment (SuperSandro2000) [inline pkgs/applications/networking/irc/senpai/default.nix:5]:
```suggestion
  version = "unstable-2021-05-27";
```

File: pkgs/applications/networking/irc/senpai/default.nix [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+  nativeBuildInputs = [
+    scdoc
+    installShellFiles
+  ];
+
+  postInstall = ''
+    scdoc < doc/senpai.1.scd > doc/senpai.1
+    scdoc < doc/senpai.5.scd > doc/senpai.5
+    installManPage doc/senpai.1
+    installManPage doc/senpai.5
```
Comment (SuperSandro2000) [inline pkgs/applications/networking/irc/senpai/default.nix:29]:
```suggestion
    installManPage doc/senpai.*
```

**Comment by malte-v** [comment]
Applied suggestions. Should I rebase or are multiple commits ok?

**Comment by SuperSandro2000** [comment]
I just squashed it.

**Pull request merged**